src/tools/chip-cert/CertUtils.cpp

```diff
@@ -842,7 +860,7 @@ bool MakeCert(CertType certType, const ToolChipDN * subjectDN, X509 * caCert, EV
               EVP_PKEY * newKey, CertStructConfig & certConfig)
 {
     bool res  = true;
-    bool isCA = (certType != CertType::kNode);
+    bool isCA = (certType != CertType::kNode && certType != CertType::kNetworkIdentity);
```

Comment:
Maybe instead use == ICAC || == RCAC. This is fine too, but it seemed odd to negatively check for things not CAs rather than explicitly looking for CA certs.

Reply:
Yeah I thought that was odd, possibly even currently buggy for creating a firmware signing cert

Reply:
Fixed.